ros_ign_gazebo_demos/launch/robot_description_publisher.launch.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
```

Comment:
Thanks for the demo. Mind adding an item for it to `ros_ign_gazebo_demos/README.md`?

Reply:
Sure I will add it.
Is the name of the launch file good? Maybe you have something better in mind?

Reply:
Yeah that sounds good to me, thanks!